tests/comp_options-textformat.js

```diff
@@ -7,21 +7,28 @@ import \"google/protobuf/descriptor.proto\";\
 message MyOptions {\
   string a = 1;\
   string b = 2;\
+  MyOptions c = 3;\
+  repeated string d = 4;\
+  repeated MyOptions e = 5;\
 }\
 extend google.protobuf.FieldOptions {\
   MyOptions my_options = 50000;\
 }\
 message Test {\
   string value = 1 [(my_options) = { a: \"foo\" b: \"bar\" }];\
-  string value2 = 2 [(my_options) = { a: \"foo\" b { c: \"bar\" } }];\
+  string value2 = 2 [(my_options) = { a: \"foo\" c { a: \"bar\" } }];\
```

Comment:
Note: I modified this test slightly so that it would parse correctly with protoc while maintaining the spirit of the original test.